flink/src/main/java/org/apache/iceberg/flink/FlinkCatalogFactory.java

```diff
@@ -52,13 +57,19 @@
 
   // Can not just use "type", it conflicts with CATALOG_TYPE.
   public static final String ICEBERG_CATALOG_TYPE = "catalog-type";
+
   public static final String HIVE_URI = "uri";
   public static final String HIVE_CLIENT_POOL_SIZE = "clients";
   public static final String HADOOP_WAREHOUSE_LOCATION = "warehouse";
 
+  public static final String HIVE_SITE_PATH = "hive-site-path";
```

Comment:
Could we align with the `HiveCatalog` in flink  by using `hive-conf-dir` ?

Reply:
ok,I will update it